pyscriptjs/src/components/pyscript.ts

```diff
@@ -71,163 +71,178 @@ export function make_PyScript(interpreter: InterpreterClient, app: PyScriptApp)
-    ['py-keypress', 'keypress'],
-    ['py-keyup', 'keyup'],
-
-    // Mouse Events
-    ['py-click', 'click'],
-    ['py-dblclick', 'dblclick'],
-    ['py-mousedown', 'mousedown'],
-    ['py-mousemove', 'mousemove'],
-    ['py-mouseout', 'mouseout'],
-    ['py-mouseover', 'mouseover'],
-    ['py-mouseup', 'mouseup'],
-    ['py-mousewheel', 'mousewheel'],
-    ['py-wheel', 'wheel'],
-
-    // Drag Events
-    ['py-drag', 'drag'],
-    ['py-dragend', 'dragend'],
-    ['py-dragenter', 'dragenter'],
-    ['py-dragleave', 'dragleave'],
-    ['py-dragover', 'dragover'],
-    ['py-dragstart', 'dragstart'],
-    ['py-drop', 'drop'],
-    ['py-scroll', 'scroll'],
-
-    // Clipboard Events
-    ['py-copy', 'copy'],
-    ['py-cut', 'cut'],
-    ['py-paste', 'paste'],
-
-    // Media Events
-    ['py-abort', 'abort'],
-    ['py-canplay', 'canplay'],
-    ['py-canplaythrough', 'canplaythrough'],
-    ['py-cuechange', 'cuechange'],
-    ['py-durationchange', 'durationchange'],
-    ['py-emptied', 'emptied'],
-    ['py-ended', 'ended'],
-    ['py-loadeddata', 'loadeddata'],
-    ['py-loadedmetadata', 'loadedmetadata'],
-    ['py-loadstart', 'loadstart'],
-    ['py-pause', 'pause'],
-    ['py-play', 'play'],
-    ['py-playing', 'playing'],
-    ['py-progress', 'progress'],
-    ['py-ratechange', 'ratechange'],
-    ['py-seeked', 'seeked'],
-    ['py-seeking', 'seeking'],
-    ['py-stalled', 'stalled'],
-    ['py-suspend', 'suspend'],
-    ['py-timeupdate', 'timeupdate'],
-    ['py-volumechange', 'volumechange'],
-    ['py-waiting', 'waiting'],
-
-    // Misc Events
-    ['py-toggle', 'toggle'],
-]);
+const browserEvents: Array<string> = new Array<string>(
+    'click',
+    'keydown',
+    'afterprint',
+    'beforeprint',
+    'beforeunload',
+    'error',
+    'hashchange',
+    'load',
+    'message',
+    'offline',
+    'online',
+    'pagehide',
+    'pageshow',
+    'popstate',
+    'resize',
+    'storage',
+    'unload',
+    'blur',
+    'change',
+    'contextmenu',
+    'focus',
+    'input',
+    'invalid',
+    'reset',
+    'search',
+    'select',
+    'submit',
+    'keydown',
+    'keypress',
+    'keyup',
+    'dblclick',
+    'mousedown',
+    'mousemove',
+    'mouseout',
+    'mouseover',
+    'mouseup',
+    'mousewheel',
+    'wheel',
+    'drag',
+    'dragend',
+    'dragenter',
+    'dragleave',
+    'dragover',
+    'dragstart',
+    'drop',
+    'scroll',
+    'copy',
+    'cut',
+    'paste',
+    'abort',
+    'canplay',
+    'canplaythrough',
+    'cuechange',
+    'durationchange',
+    'emptied',
+    'ended',
+    'loadeddata',
+    'loadedmetadata',
+    'loadstart',
+    'pause',
+    'play',
+    'playing',
+    'progress',
+    'ratechange',
+    'seeked',
+    'seeking',
+    'stalled',
+    'suspend',
+    'timeupdate',
+    'volumechange',
+    'waiting',
+    'toggle',
+);
 
 /** Initialize all elements with py-* handlers attributes  */
 export async function initHandlers(interpreter: InterpreterClient) {
```

Comment:
I don't personally care about other linters errors but this one in particular seems to be reasonable ... why is this callback `async` if there's not a single `await` in it?

Let's remember that if consumer code does `await initHandlers(...)` works regardless the callback is a Promise (*async*) or not, so this `async` feels a bit off to me ... what do you think? 🤔